packages/keycloak-user-management/src/components/HeaderBreadCrumb/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Tabs } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+
+const { TabPane } = Tabs;
+
+interface HeaderBreadCrumbProps {
+  isAdmin?: boolean;
+  userId?: string;
+}
+
+export const HeaderBreadCrumb: React.FC<HeaderBreadCrumbProps> = (props: HeaderBreadCrumbProps) => {
+  const { userId } = props;
+  const [activeKey, setActiveKey] = React.useState<string>('');
+  const handleTabLink = (key: string) => {
+    if (key === 'credentials') {
+      history.push(`/user/credentials/${userId}`);
+    } else if (key === 'details') {
+      history.push(`/user/edit/${userId}`);
+    } else {
+      history.push(`/${key}`);
```

Comment:
- Must we use history.push?  Do we not have a router package?
- Please, please do not turn to magic strings so liberally

Reply:
@kelvin-muchiri was this one addressed?